src/lib/jsonParseV2/parseSubscription.cpp

```diff
@@ -217,7 +217,7 @@ std::string parseSubscription(ConnectionInfo* ciP, SubscriptionUpdate* subsP, bo
   {
```

Comment:
An entry about this PR should be added to CHANGES_NEXT_RELASE file (citing also the issue). Have a look to existing entries in that file and use the same style.

Reply:
fixed in 442ff26